jumanji/training/configs/config.yaml

```
@@ -1,12 +1,12 @@
 defaults:
     - _self_
-    - env: snake  # [bin_pack, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, maze, minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp]
+    - env: snake  # [snake, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, maze, minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp]
 
 agent: random  # [random, a2c]
 
 seed: 0
 
 logger:
-    type: terminal  # [neptune, tensorboard, terminal]
+    type: neptune  # [neptune, tensorboard, terminal]
```

Please revert back to previous configs
```suggestion
    type: terminal  # [neptune, tensorboard, terminal]
```